app/src/main/res/layout/activity_basic.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
     tools:context=".BasicActivity">
 
     <Button
         android:id="@+id/button_request"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Request" />
+        android:text="Request"
+        tools:ignore="HardcodedText"/>
```

Comment:
I didn't do that because `BasicActivity` isn't part of the sample so it shouldn't pollute the `strings.xml` file. Do you agree with this or should I add it to the strings file anyway?